datasource/result_constants.py

```diff
@@ -0,0 +1,129 @@
+from collections import namedtuple
+
+# NetCDF variable information
+NcVar = namedtuple("NcVar", ["name", "verbose_name", "unit"])
```

Comment:
Ik heb zelf namedtuples nog nooit gebruikt ("ik maak wel een class of een dict"), maar op deze manier zien ze er toch wel èrg makkelijk uit.
Hoog tijd om ze zelf ook te gaan gebruiken :-)